docs/tests.md

```diff
@@ -84,7 +84,17 @@ testing portions of your code that can be tested without relying on a multiworld
 
 ## Running Tests
 
-In PyCharm, running all tests can be done by right-clicking the root `test` directory and selecting `run Python tests`.
-If you do not have pytest installed, you may get import failures. To solve this, edit the run configuration, and set the
-working directory of the run to the Archipelago directory. If you only want to run your world's defined tests, repeat
-the steps for the test directory within your world.
+#### Using Pycharm
+In PyCharm, running all tests can be done by right-clicking the root test directory and selecting Run 'Archipelago Unittests'. 
+If you do not have pytest installed, you may get import failures. To solve this, edit the run configuration, 
```

Comment:
`pytest` is not required to run tests in the IDE, so this wording still sounds odd.

Something like "unless you configured pycharm to use pytest as a test runner, ..." would be more correct.